mage/template.go

```diff
@@ -6,12 +6,17 @@ var tpl = `// +build ignore
 package main
 
 import (
+	"context"
 	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
+
 	"strings"
+
```

Comment:
it doesn't really matter, but it would be nice to clean up these extra blank lines